src/igtcloud/client/tools/upload_project.py

```diff
@@ -83,6 +87,80 @@ def upload_project(local_folder: str, project_name: str, institute_name: str = N
                             f"files_skipped: {len(files_skipped)}")
 
 
+def upload_annotation_files_private_method(institutes, local_folder, max_workers_files):
```

Comment:
Please rename this function to `upload_annotation_files`. Python does not have a private access modifier, and it is fine to publicly call this method (like the other ones in this file).

Reply:
Changed method name to upload_annotation_files